mapbox_maps_flutter_mobile/lib/src/mapbox_map.dart

```diff
@@ -344,10 +344,16 @@ class MapboxMap extends ChangeNotifier {
   /// The map will retain its current values for any details not passed via the camera options argument.
   /// It is not guaranteed that the provided `camera options` will be set, the map may apply constraints resulting in a
   /// different `camera state`.
-  Future<void> setCamera(CameraOptions cameraOptions) =>
-      _cameraManager.setCamera(cameraOptions);
+  @override
+  Future<void> setCamera(CameraOptions cameraOptions) {
+    if (cameraOptions.center != null) {
+      cameraOptions.center = Point._from(cameraOptions.center!);
```

Comment:
Why does this have to be re-assigned?

Reply:
initially I left the part where we extend turf's Point in mobile package, that's why I had to cast type here so the Codec would understand, otherwise it would throw an error Invalid type. But now I moved those extensions to interface package and removed this cast